site/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "private": true,
   "devDependencies": {
-    "gulp": "^3.8.8"
+    "gulp": "^3.8.8",
+    "gulp-util": "^3.0.7"
```

Comment:
Add `gulp-util` as an explicit dependency in case Elixir's dependencies ever change.